test/e2e/plain_provisioner_test.go

```diff
@@ -345,6 +345,70 @@ var _ = Describe("plain provisioner bundle", func() {
 			))
 		})
 	})
+	When("a bundle containing nested directry is created", func() {
+		var (
+			bundle *rukpakv1alpha1.Bundle
+			ctx    context.Context
+		)
+		BeforeEach(func() {
+			ctx = context.Background()
+
+			By("creating the testing Bundle resource")
+			bundle = &rukpakv1alpha1.Bundle{
+				ObjectMeta: metav1.ObjectMeta{
+					GenerateName: "olm-crds-unsupported",
+				},
+				Spec: rukpakv1alpha1.BundleSpec{
+					ProvisionerClassName: plainProvisionerID,
+					Source: rukpakv1alpha1.BundleSource{
+						Type: rukpakv1alpha1.SourceTypeImage,
+						Image: &rukpakv1alpha1.ImageSource{
+							Ref: "testdata/bundles/plain-v0:subdir",
+						},
+					},
+				},
+			}
+			err := c.Create(ctx, bundle)
+			Expect(err).To(BeNil())
+		})
+		AfterEach(func() {
+			By("deleting the testing Bundle resource")
+			err := c.Delete(ctx, bundle)
+			Expect(err).To(BeNil())
+		})
+		It("should eventually report a successful state", func() {
+			By("eventually reporting an Unpacked phase", func() {
+				Eventually(func() (string, error) {
+					if err := c.Get(ctx, client.ObjectKeyFromObject(bundle), bundle); err != nil {
+						return "", err
+					}
+					return bundle.Status.Phase, nil
```

Comment:
IMO, this should actually fail (and we should tighten up the spec and code to call this out as invalid).

Ignoring subdirectories will be a poor UX.

Reply:
I agree.  It should be handled in https://github.com/operator-framework/rukpak/issues/159

Reply:
#159 says we should support nested directories, but IMO, there's no reason to do that until there's a specific use case where nested directories are the most logical solution.  

> we should tighten up the spec and code to call [nested manifests directories] out as invalid

Do we need another issue that talks about that?

Reply:
I thought #195 is only for the e2e test.  I'll include the manifest reader change in this PR.  Thanks!